tx/textual/valuerenderer/valuerenderer_test.go

```diff
@@ -116,6 +117,14 @@ func TestGetADR050ValueRenderer(t *testing.T) {
 	}
 }
 
+func TestTimestampDispatch(t *testing.T) {
```

Comment:
This dispatcher test was the initial intent of `TestGetADR050ValueRenderer`. I think once we make all value renderers public, we can merge this test with `TestGetADR050ValueRenderer` (rename it to sthg better), and use `require.IsType` as you did here.

I can take care of this in a follow-up PR.

Reply:
Ack.